### PR TITLE
ci: Fix MSVC 2019 compatibility by skipping testArgsParser inside Meson

### PR DESCRIPTION
**Pull request by amyspark**
:wave:

Second PR of last night's CI upgrade branch.

### CODE REVIEW

File: test/crunch++/meson.build [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
 
+if cxx.get_id() == 'msvc' or cxx.version().version_compare('< 19.30.30704')
+	# MSVC 2019 crashes on testArgsParser
+	# https://developercommunity.visualstudio.com/t/Crash-on-constexpr-creation-of-a-std::ar/1547919
+	warning('Skipping testArgsParser test as it crashes MSVC')
+else
+	libCrunchppTestsNorm += ['testArgsParser']
+endif
+
+message('MSVC version: @0@'.format(cxx.version()))
```
Comment (dragonmux) [inline test/crunch++/meson.build:13]: this should probably be gated behind `if isMSVC` (and that constant propagated through to here).

**Review by dragonmux — CHANGES_REQUESTED** [review]
Ah, excellent call. There's one small item we spotted in review and once that's addressed and the PR rebased on main, we're happy to merge.

File: test/crunch++/meson.build (hunk 1 above)
Comment (amyspark) [reply test/crunch++/meson.build]: I had that change done as part of the MSYS fix, so I'll backport it now.

**Review by dragonmux — APPROVED** [review]
LGTM! Thank you. Merging

**Pull request merged**